flexmeasures/ui/utils/view_utils.py

```diff
@@ -86,14 +87,19 @@ def render_flexmeasures_template(html_filename: str, **variables):
         options["downloadFileName"] = f"asset-{asset.id}-{asset.name}"
     variables["chart_options"] = json.dumps(options)
 
-    variables["menu_logo"] = current_app.config.get("FLEXMEASURES_MENU_LOGO_PATH")
+    account: Account | None = (
+        current_user.account if current_user.is_authenticated else None
+    )
+
+    # check if user has logo_url set
+    if account and account.logo_url:
```

Comment:
Here, we should have similar logic as with the colors: If the consultancy account has set this, (and I haven't), take theirs.

Reply:
ok i can implement that